pkgs/tools/misc/powerline-rs/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, lib, rustPlatform, fetchFromGitHub, pkgconfig, file, perl, curl, cmake, openssl_1_1_0, libssh2, libgit2, libzip, Security }:
+{ stdenv, lib, rustPlatform, fetchFromGitHub, pkgconfig, file, perl, curl, cmake, openssl_1_1, libssh2, libgit2, libzip, Security }:
```

Comment:
Could you use `openssl` here, and call `powerline-rs` with `openssl = openssl_1_1` in `all-packages.nix`?